doc/generated/libpmemfile-posix.3

```diff
@@ -36,13 +36,10 @@
 \f[B]libpmemfile\-posix\f[] \- user space persistent memory aware file
 system API
 .SH SYNOPSIS
-.IP
-.nf
-\f[C]
-#include\ <libpmemfile\-posix.h>
-cc\ \-std=gnu99\ ..\ \-lpmemfile\-posix\ \-lpmemobj
-\f[]
-.fi
+.PP
+\f[C]#include\ <libpmemfile\-posix.h>\f[]
+.PP
+\f[C]cc\ \-std=gnu99\ ..\ `pkg\-config\ libpmemfile\-posix\ \-\-cflags\ \-\-libs`\f[]
```

Comment:
Nope. With a simple fix (see the last commit) it's even possible to build with C90, but I don't think we want to go as far. Fixed.